src/components/Key.tsx

```diff
@@ -0,0 +1,79 @@
+import { useCallback, useState, useEffect, useMemo } from "react";
+import { Api } from "../utils/Api";
+import { Button } from "./Button";
+
+import cn from "classnames";
+
+interface KeyProps {
+  value: string;
```

Comment:
можно сделать


```suggestion
  value: KeyType['value'];
```

Но в целом не критично